imap_processing/glows/l1a/glows_l1a.py

```diff
@@ -133,35 +133,35 @@ def generate_de_dataset(
 
     # Each DirectEventL1A class covers 1 second of direct events data
     direct_events = np.zeros((len(de_l1a_list), len(de_l1a_list[0].direct_events), 4))
+    missing_packets_sequence = ""
 
     support_data: dict = {
-        # "flight_software_version": [], # breaks
-        "seq_count_in_pkts_file": [],  # works
-        "number_of_de_packets": [],  # works
-        # "missing_packet_sequences": [] # breaks
+        # "flight_software_version": [],
+        "seq_count_in_pkts_file": [np.uint16, []],
+        "number_of_de_packets": [np.uint32, []],
     }
 
     data_every_second: dict = {
```

Comment:
Add a Comment explaining that the 0th element of each dict value is the datatype of the output xr.DataArray?